src/pages/index.js

```diff
@@ -71,22 +104,35 @@ imageModal.setEventListeners();
 
 // #region Cards rendering
 
-const cards = document.querySelector(".cards");
+// Is there a better practice for defining a global variable inside a promise resolve?
```

Comment:
**Could be improved**

The best way is to remake `renderItems` so that it could accept the array of cards


```js
 cardsSection.renderItems(cards);

```

Then you can define the `cardsSection` here


```js
// Is there a better practice for defining a global variable inside a promise resolve?
const cardsSection = new Section(
  {
    renderer: (data) => {
      const card = getCardElement(data);
      cardsSection.addItem(card);
    },
  },
  ".cards"
);

api.getCards().then((res) => {
  const cards = Array.from(res).reverse();
  cardsSection.renderItems(cards);

```